modules/skybrowser/src/utility.cpp

```diff
@@ -150,11 +167,10 @@ bool isCoordinateInView(const glm::dvec3& equatorial) {
     // Check if image coordinate is within current FOV
     glm::dvec3 localCamera = equatorialToLocalCamera(equatorial);
     glm::dvec3 coordsScreen = localCameraToScreenSpace3d(localCamera);
-    double r = static_cast<float>(windowRatio());
-
-    bool isCoordInView = abs(coordsScreen.x) < r && abs(coordsScreen.y) < 1.f &&
-                         coordsScreen.z < 0;
+    double r = windowRatio();
 
+    bool isCoordInView =
+        abs(coordsScreen.x) < r && abs(coordsScreen.y) < 1.f && coordsScreen.z < 0;
```

Comment:
`coordsScreen.z < 0.f` ? Don't remember if we do an exception for 0

Reply:
No, we don't... you're right